www/src/views/starter-library/filtered-starters.js

```diff
@@ -101,7 +101,7 @@ export default class FilteredStarterLibrary extends Component {
 
     return (
       <section className="showcase" css={{ display: `flex` }}>
-        <SidebarContainer>
+        <SidebarContainer overrideCSS={{ overflowY: "auto" }}>
```

Comment:
```suggestion
        <SidebarContainer css={{ overflowY: "auto" }}>
```

#6649 doesn't seem relevant anymore because we've moved off of glamor?

Reply:
I believe the problem OP of #6649 had is still relevant, because `css` is [syntactic sugar](https://emotion.sh/docs/css#css-prop) for `className={css(...)}`, so we can't pass it like a prop without thinking it's actually `className`.

I'm gonna refactor it right now to use `css` and `className`.

Reply:
I'm not really sure of your point here, but it looks like you went this route?